pages/index.js

```diff
@@ -10,20 +9,7 @@ const Wrapper = styled.div`
 
 const PageTitle = styled.h1`
   text-align: center;
-  margin: 30px 0;
-`;
-
-const Nav = styled.nav`
-  display: flex;
-  gap: 30px;
-`;
-
-const NavLink = styled(Link)`
-  text-decoration: none;
-  color: navy;
-  &:hover {
-    text-decoration: underline;
-  }
+  margin: 30px auto 10px auto;
```

Comment:
A very petty hint. And actually only to show that a shorthand is be possible. 
```suggestion
 margin: 30px auto 10px;
```